metricbeat/module/kubernetes/util/kubernetes_test.go

```diff
@@ -59,6 +60,23 @@ func TestBuildMetadataEnricher(t *testing.T) {
 		{"name": "unknown"},
 		{
 			"name":    "enrich",
+			"_module": common.MapStr{"label": "value", "pod": common.MapStr{"name": "enrich", "uid": "mockuid"}},
+		},
+	}, events)
+
+	// Enrich a pod (metadata goes in root level)
+	events = []common.MapStr{
+		{"name": "unknown"},
+		{"name": "enrich"},
+	}
+	enricher.isPod = true
+	enricher.Enrich(events)
+
+	assert.Equal(t, []common.MapStr{
+		{"name": "unknown"},
+		{
+			"name":    "enrich",
+			"uid":     "mockuid",
```

Comment:
Do we also need to update `data.json` file of `pod` and `state_pod` metricsets?

Reply:
pushed e17c704